config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root "users#index"
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show, :create, :new] do
+      resources :comments, only: [:create]
```

Comment:
- [ ] Based on the requirements of this project,  there should be a pair of routes for creating comments:

`resources :comments, only: [:create, :new]`


Let's kindly fix :wrench: it.


- [ ] In order for us to create a new comment, there should be a form in the views/comments/new.html.erb file which accept inputs for creating comments. Let's kindly fix :wrench: it.

Please this should not be done in the views/post/ directory. I saw a partial(_add_comment.html.erb) here for creating a new comment. This should be done properly in the directory shown above. This is a Rails great way of organizing our codebase for future reference.

Reply:
Thank sir i really appreciate you for taking much of your time to view my code ,great job and all the changes required have been fixed .